hw2/bor/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using bor;
+class Program
+{
+    static void Main(string[] args)
+    {
+        Tree tree = new Tree();
+        var continueProgram = true;
+        while (continueProgram)
+        {
+            Console.WriteLine("Что вы хотите сделать с бором?\n0 - выйти из программы;\n1 - добавить слово в бор;\n2 - проверить, содержется ли " +
+                "данное слово в боре;\n3 - удалить слово из бора;\n4 - узнать, сколько есть слов с заданным префиксом;" +
+                "\n5 - узнать размер бора.");
```

Comment:
```suggestion
            Console.WriteLine("Что вы хотите сделать с бором?\n"
                + "0 - выйти из программы;\n
                + "1 - добавить слово в бор;\n
                + "2 - проверить, содержется ли данное слово в боре;\n"
                + "3 - удалить слово из бора;\n"
                + "4 - узнать, сколько есть слов с заданным префиксом;\n" 
                + "5 - узнать размер бора.");
```
Так просто удобнее в разы